### ISSUE TITLE
RemoveUnusedImports recipe removes imports used in javadoc

### ISSUE THREAD

**Issue by murdos** [openrewrite/rewrite#628]
## Example: 
```java
import org.springframework.dao.DuplicateKeyException;

public interface SettingsDAO {

     /*
     * @throws DuplicateKeyException if settings are already inserted
     */
    void updateSettings(String settings);
}
```

will become
```java
public interface SettingsDAO {

     /*
     * @throws DuplicateKeyException if settings are already inserted
     */
    void updateSettings(String settings);
}
```

Same issue with javadoc using `{@link SomeClass}`

**Comment by sambsnyd** [comment]
Good catch, thanks for reporting!

**Comment by jkschneider** [comment]
Support for this is included in #921 , so going to close this as a duplicate.
